k4FWCore/components/IOSvc.h

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2014-2024 Key4hep-Project.
+ *
+ * This file is part of Key4hep.
+ * See https://key4hep.github.io/key4hep-doc/ for further info.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef FWCORE_IOSVC_H
+#define FWCORE_IOSVC_H
+
+#include "Gaudi/Property.h"
+#include "GaudiKernel/IDataProviderSvc.h"
+#include "GaudiKernel/IHiveWhiteBoard.h"
+#include "GaudiKernel/IIncidentListener.h"
+#include "GaudiKernel/IIncidentSvc.h"
+#include "GaudiKernel/Service.h"
+
+#include "podio/ROOTReader.h"
+#include "podio/ROOTWriter.h"
+
+#include "k4FWCore/KeepDropSwitch.h"
+
+#include "IIOSvc.h"
```

Comment:
This doesn't work, we don't have in the include directories the top level directory or components, only what's in `include`. I prefer to keep it as it is, since we don't need to add more directories for components.

Reply:
Ok, if [I]IOSvc.h are not installed publicly